package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "liveperson-functions-client",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
It's not a patch it is actually a none breaking change, that introduces a new feature

Reply:
updated